src/tests/ftest/rebuild/delete_objects.py

```diff
@@ -90,7 +86,10 @@ def test_rebuild_delete_objects(self):
         Use Cases:
             foo
 
-        :avocado: tags=all,large,full_regression,rebuild,rebuilddeleteobject
+        :avocado: tags=all,daily_regression
+        :avocado: tags=vm,large
+        :avocado: tags=rebuild
+        :avocado: tags=pool,rebuild_tests,rebuilddeleteobject
```

Comment:
can you please change `rebuild_tests` to `deleteobjects` as something similar is already being used under rebuild/rebuild_tests.py